cli/io/fio.py

```diff
@@ -0,0 +1,114 @@
+import itertools
+
+from cli import Cli
+from cli.utilities.packages import Package
+
+
+class FioError(Exception):
```

Comment:
Change this exception to `IOEroro` and move to - https://github.com/red-hat-storage/cephci/blob/master/cli/exceptions.py